Rakefile

```diff
@@ -113,6 +119,25 @@ rescue
   puts 'Rubocop not loaded'
 end
 
+namespace :config do
```

Comment:
The idea of this is to occasionally 'sync' the files in git with the migrations? Sort of like keeping a database schema in git for your Rails application?